--- sdrplay/install_lib.sh
+++ sdrplay/install_lib_patched.sh
@@ -3,19 +3,7 @@
 
 echo "Installing SDRplay RSP API library 2.13..."
 
-more sdrplay_license.txt
-
-while true; do
-    echo "Press y and RETURN to accept the license agreement and continue with"
-    read -p "the installation, or press n and RETURN to exit the installer [y/n] " yn
-    case $yn in
-        [Yy]* ) break;;
-        [Nn]* ) exit;;
-        * ) echo "Please answer y or n";;
-    esac
-done
-
-export ARCH=`arch`
+export ARCH=`uname -m`
 export VERS="2.13"
 
 echo "Architecture: ${ARCH}"
@@ -60,16 +48,6 @@
     echo "ERROR: udev rules directory not found, add udev support and run the"
     echo "installer again. udev support can be added by running..."
     echo "sudo apt-get install libudev-dev"
-    echo " "
-    exit 1
-fi
-
-if /sbin/ldconfig -p | /bin/fgrep -q libusb-1.0; then
-    echo "Libusb found, continuing..."
-else
-    echo " "
-    echo "ERROR: Libusb cannot be found. Please install libusb and then run"
-    echo "the installer again. Libusb can be installed from http://libusb.info"
     echo " "
     exit 1
 fi
