--- sdrplay/install_lib.sh	2018-06-21 20:47:08.000000000 +0200
+++ sdrplay/install_lib_patched.sh	2019-01-22 17:35:57.761927708 +0100
@@ -3,18 +3,6 @@
 
 echo "Installing SDRplay RSP API library 2.13..."
 
-more sdrplay_license.txt
-
-while true; do
-    echo "Press y and RETURN to accept the license agreement and continue with"
-    read -p "the installation, or press n and RETURN to exit the installer [y/n] " yn
-    case $yn in
-        [Yy]* ) break;;
-        [Nn]* ) exit;;
-        * ) echo "Please answer y or n";;
-    esac
-done
-
 export ARCH=`arch`
 export VERS="2.13"
 
@@ -63,16 +51,6 @@
     echo " "
     exit 1
 fi
-
-if /sbin/ldconfig -p | /bin/fgrep -q libusb-1.0; then
-    echo "Libusb found, continuing..."
-else
-    echo " "
-    echo "ERROR: Libusb cannot be found. Please install libusb and then run"
-    echo "the installer again. Libusb can be installed from http://libusb.info"
-    echo " "
-    exit 1
-fi
 
 sudo ldconfig
 
