--- CMakeLists.txt.orig	2021-09-28 14:33:14.329598412 +0200
+++ CMakeLists.txt	2021-09-28 14:34:23.052345270 +0200
@@ -106,24 +106,6 @@
 
 
 #
-# build and install hamlib locally so it can be referenced by the
-# WSJT-X build
-#
-ExternalProject_Add (hamlib
-  GIT_REPOSITORY ${hamlib_repo}
-  GIT_TAG ${hamlib_TAG}
-  GIT_SHALLOW False
-  URL ${CMAKE_CURRENT_SOURCE_DIR}/src/${__hamlib_upstream}.tar.gz
-  URL_HASH MD5=${hamlib_md5sum}
-  #UPDATE_COMMAND ${CMAKE_COMMAND} -E env "[ -f ./bootstrap ] && ./bootstrap"
-  PATCH_COMMAND ${PATCH_EXECUTABLE} -p1 -N < ${CMAKE_CURRENT_SOURCE_DIR}/hamlib.patch
-  CONFIGURE_COMMAND <SOURCE_DIR>/configure --prefix=<INSTALL_DIR> --disable-shared --enable-static --without-cxx-binding ${EXTRA_FLAGS} # LIBUSB_LIBS=${USB_LIBRARY}
-  BUILD_COMMAND $(MAKE) all V=1 # $(MAKE) is ExternalProject_Add() magic to do recursive make
-  INSTALL_COMMAND $(MAKE) install-strip V=1 DESTDIR=""
-  STEP_TARGETS update install
-  )
-
-#
 # custom target to make a hamlib source tarball
 #
 add_custom_target (hamlib_sources
@@ -161,7 +143,6 @@
 # build and optionally install WSJT-X using the hamlib package built
 # above
 #
-ExternalProject_Get_Property (hamlib INSTALL_DIR)
 ExternalProject_Add (wsjtx
   GIT_REPOSITORY ${wsjtx_repo}
   GIT_TAG ${WSJTX_TAG}
@@ -186,14 +167,8 @@
   DEPENDEES build
   )
 
-set_target_properties (hamlib PROPERTIES EXCLUDE_FROM_ALL 1)
 set_target_properties (wsjtx PROPERTIES EXCLUDE_FROM_ALL 1)
 
-add_dependencies (wsjtx-configure hamlib-install)
-add_dependencies (wsjtx-build hamlib-install)
-add_dependencies (wsjtx-install hamlib-install)
-add_dependencies (wsjtx-package hamlib-install)
-
 # export traditional targets
 add_custom_target (build ALL DEPENDS wsjtx-build)
 add_custom_target (install DEPENDS wsjtx-install)
