diff -ur wsjtx-orig/CMakeLists.txt wsjtx/CMakeLists.txt
--- wsjtx-orig/CMakeLists.txt	2023-01-28 17:43:05.586124507 +0100
+++ wsjtx/CMakeLists.txt	2023-01-28 17:56:07.108634912 +0100
@@ -122,7 +122,7 @@
 option (WSJT_QDEBUG_TO_FILE "Redirect Qt debuging messages to a trace file.")
 option (WSJT_SOFT_KEYING "Apply a ramp to CW keying envelope to reduce transients." ON)
 option (WSJT_SKIP_MANPAGES "Skip *nix manpage generation.")
-option (WSJT_GENERATE_DOCS "Generate documentation files." ON)
+option (WSJT_GENERATE_DOCS "Generate documentation files.")
 option (WSJT_RIG_NONE_CAN_SPLIT "Allow split operation with \"None\" as rig.")
 option (WSJT_TRACE_UDP "Debugging option that turns on UDP message protocol diagnostics.")
 option (WSJT_BUILD_UTILS "Build simulators and code demonstrators." ON)
@@ -170,77 +170,7 @@
   )
 
 set (wsjt_qt_CXXSRCS
-  helper_functions.cpp
-  qt_helpers.cpp
-  widgets/MessageBox.cpp
-  MetaDataRegistry.cpp
-  Network/NetworkServerLookup.cpp
   revision_utils.cpp
-  L10nLoader.cpp
-  WFPalette.cpp
-  Radio.cpp
-  RadioMetaType.cpp
-  NonInheritingProcess.cpp
-  models/IARURegions.cpp
-  models/Bands.cpp
-  models/Modes.cpp
-  models/FrequencyList.cpp
-  models/StationList.cpp
-  widgets/FrequencyLineEdit.cpp
-  widgets/FrequencyDeltaLineEdit.cpp
-  item_delegates/CandidateKeyFilter.cpp
-  item_delegates/ForeignKeyDelegate.cpp
-  item_delegates/MessageItemDelegate.cpp
-  validators/LiveFrequencyValidator.cpp
-  GetUserId.cpp
-  Audio/AudioDevice.cpp
-  Transceiver/Transceiver.cpp
-  Transceiver/TransceiverBase.cpp
-  Transceiver/EmulateSplitTransceiver.cpp
-  Transceiver/TransceiverFactory.cpp
-  Transceiver/PollingTransceiver.cpp
-  Transceiver/HamlibTransceiver.cpp
-  Transceiver/HRDTransceiver.cpp
-  Transceiver/DXLabSuiteCommanderTransceiver.cpp
-  Network/NetworkMessage.cpp
-  Network/MessageClient.cpp
-  widgets/LettersSpinBox.cpp
-  widgets/HintedSpinBox.cpp
-  widgets/RestrictedSpinBox.cpp
-  widgets/HelpTextWindow.cpp
-  SampleDownloader.cpp
-  SampleDownloader/DirectoryDelegate.cpp
-  SampleDownloader/Directory.cpp
-  SampleDownloader/FileNode.cpp
-  SampleDownloader/RemoteFile.cpp
-  DisplayManual.cpp
-  MultiSettings.cpp
-  validators/MaidenheadLocatorValidator.cpp
-  validators/CallsignValidator.cpp
-  widgets/SplashScreen.cpp
-  EqualizationToolsDialog.cpp
-  widgets/DoubleClickablePushButton.cpp
-  widgets/DoubleClickableRadioButton.cpp
-  Network/LotWUsers.cpp
-  models/DecodeHighlightingModel.cpp
-  widgets/DecodeHighlightingListView.cpp
-  models/FoxLog.cpp
-  widgets/AbstractLogWindow.cpp
-  widgets/FoxLogWindow.cpp
-  widgets/CabrilloLogWindow.cpp
-  item_delegates/CallsignDelegate.cpp
-  item_delegates/MaidenheadLocatorDelegate.cpp
-  item_delegates/FrequencyDelegate.cpp
-  item_delegates/FrequencyDeltaDelegate.cpp
-  item_delegates/SQLiteDateTimeDelegate.cpp
-  models/CabrilloLog.cpp
-  logbook/AD1CCty.cpp
-  logbook/WorkedBefore.cpp
-  logbook/Multiplier.cpp
-  Network/NetworkAccessManager.cpp
-  widgets/LazyFillComboBox.cpp
-  widgets/CheckableItemComboBox.cpp
-  widgets/BandComboBox.cpp
   )
 
 set (wsjt_qtmm_CXXSRCS
@@ -1089,9 +1019,6 @@
 if (WSJT_GENERATE_DOCS)
   add_subdirectory (doc)
 endif (WSJT_GENERATE_DOCS)
-if (EXISTS ${CMAKE_SOURCE_DIR}/tests AND IS_DIRECTORY ${CMAKE_SOURCE_DIR}/tests)
-  add_subdirectory (tests)
-endif ()
 
 # build a library of package functionality (without and optionally with OpenMP support)
 add_library (wsjt_cxx STATIC ${wsjt_CSRCS} ${wsjt_CXXSRCS})
@@ -1357,10 +1284,7 @@
 add_library (wsjt_qt STATIC ${wsjt_qt_CXXSRCS} ${wsjt_qt_GENUISRCS} ${GENAXSRCS})
 # set wsjtx_udp exports to static variants
 target_compile_definitions (wsjt_qt PUBLIC UDP_STATIC_DEFINE)
-target_link_libraries (wsjt_qt Hamlib::Hamlib Boost::log qcp Qt5::Widgets Qt5::Network Qt5::Sql)
-if (WIN32)
-  target_link_libraries (wsjt_qt Qt5::AxContainer Qt5::AxBase)
-endif (WIN32)
+target_link_libraries (wsjt_qt Qt5::Core)
 
 # build a library of package Qt functionality used in Fortran utilities
 add_library (fort_qt STATIC ${fort_qt_CXXSRCS})
@@ -1425,90 +1349,6 @@
   add_subdirectory (map65)
 endif ()
 
-# build the main application
-generate_version_info (wsjtx_VERSION_RESOURCES
-  NAME wsjtx
-  BUNDLE ${PROJECT_BUNDLE_NAME}
-  ICON ${WSJTX_ICON_FILE}
-  )
-
-add_executable (wsjtx MACOSX_BUNDLE
-  ${wsjtx_CXXSRCS}
-  ${wsjtx_GENUISRCS}
-  ${WSJTX_ICON_FILE}
-  ${wsjtx_RESOURCES_RCC}
-  ${wsjtx_VERSION_RESOURCES}
-  )
-
-if (WSJT_CREATE_WINMAIN)
-  set_target_properties (wsjtx PROPERTIES WIN32_EXECUTABLE ON)
-endif (WSJT_CREATE_WINMAIN)
-
-set_target_properties (wsjtx PROPERTIES
-  MACOSX_BUNDLE_INFO_PLIST "${CMAKE_CURRENT_SOURCE_DIR}/Darwin/Info.plist.in"
-  MACOSX_BUNDLE_INFO_STRING "${PROJECT_DESCRIPTION}"
-  MACOSX_BUNDLE_ICON_FILE "${WSJTX_ICON_FILE}"
-  MACOSX_BUNDLE_BUNDLE_VERSION ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}
-  MACOSX_BUNDLE_SHORT_VERSION_STRING "v${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}"
-  MACOSX_BUNDLE_LONG_VERSION_STRING "Version ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}${SCS_VERSION_STR}"
-  MACOSX_BUNDLE_BUNDLE_NAME "${PROJECT_BUNDLE_NAME}"
-  MACOSX_BUNDLE_BUNDLE_EXECUTABLE_NAME "${PROJECT_NAME}"
-  MACOSX_BUNDLE_COPYRIGHT "${PROJECT_COPYRIGHT}"
-  MACOSX_BUNDLE_GUI_IDENTIFIER "org.k1jt.wsjtx"
-  )
-
-target_include_directories (wsjtx PRIVATE ${FFTW3_INCLUDE_DIRS})
-if ((NOT ${OPENMP_FOUND}) OR APPLE)
-  target_link_libraries (wsjtx wsjt_fort)
-else ()
-  target_link_libraries (wsjtx wsjt_fort_omp)
-  if (OpenMP_C_FLAGS)
-    set_target_properties (wsjtx PROPERTIES
-      COMPILE_FLAGS "${OpenMP_C_FLAGS}"
-      LINK_FLAGS "${OpenMP_C_FLAGS}"
-      )
-  endif ()
-  set_target_properties (wsjtx PROPERTIES
-    Fortran_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/fortran_modules_omp
-    )
-  if (WIN32)
-    set_target_properties (wsjtx PROPERTIES
-      LINK_FLAGS -Wl,--stack,0x1000000,--heap,0x20000000
-      )
-  endif ()
-endif ()
-target_link_libraries (wsjtx Qt5::SerialPort wsjt_cxx wsjt_qt wsjt_qtmm ${FFTW3_LIBRARIES} ${LIBM_LIBRARIES})
-
-# make a library for WSJT-X UDP servers
-# add_library (wsjtx_udp SHARED ${UDP_library_CXXSRCS})
-add_library (wsjtx_udp-static STATIC ${UDP_library_CXXSRCS})
-#target_include_directories (wsjtx_udp
-#  INTERFACE
-#  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/wsjtx>
-#  )
-target_include_directories (wsjtx_udp-static
-  INTERFACE
-  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/wsjtx>
-  )
-#set_target_properties (wsjtx_udp PROPERTIES
-#  PUBLIC_HEADER "${UDP_library_HEADERS}"
-#  )
-set_target_properties (wsjtx_udp-static PROPERTIES
-  OUTPUT_NAME wsjtx_udp
-  )
-target_compile_definitions (wsjtx_udp-static PUBLIC UDP_STATIC_DEFINE)
-target_link_libraries (wsjtx_udp-static Qt5::Network Qt5::Gui)
-generate_export_header (wsjtx_udp-static BASE_NAME udp)
-
-generate_version_info (udp_daemon_VERSION_RESOURCES
-  NAME udp_daemon
-  BUNDLE ${PROJECT_BUNDLE_NAME}
-  ICON ${WSJTX_ICON_FILE}
-  FILE_DESCRIPTION "Example WSJT-X UDP Message Protocol daemon"
-  )
-add_executable (udp_daemon UDPExamples/UDPDaemon.cpp ${udp_daemon_VERSION_RESOURCES})
-target_link_libraries (udp_daemon wsjtx_udp-static)
-
 generate_version_info (wsjtx_app_version_VERSION_RESOURCES
   NAME wsjtx_app_version
   BUNDLE ${PROJECT_BUNDLE_NAME}
@@ -1518,47 +1358,9 @@
 add_executable (wsjtx_app_version AppVersion/AppVersion.cpp ${wsjtx_app_version_VERSION_RESOURCES})
 target_link_libraries (wsjtx_app_version wsjt_qt)
 
-generate_version_info (message_aggregator_VERSION_RESOURCES
-  NAME message_aggregator
-  BUNDLE ${PROJECT_BUNDLE_NAME}
-  ICON ${WSJTX_ICON_FILE}
-  FILE_DESCRIPTION "Example WSJT-X UDP Message Protocol application"
-  )
-add_resources (message_aggregator_RESOURCES /qss ${message_aggregator_STYLESHEETS})
-configure_file (UDPExamples/message_aggregator.qrc.in message_aggregator.qrc @ONLY)
-qt5_add_resources (message_aggregator_RESOURCES_RCC
-  ${CMAKE_CURRENT_BINARY_DIR}/message_aggregator.qrc
-  contrib/QDarkStyleSheet/qdarkstyle/style.qrc
-  )
-add_executable (message_aggregator
-  ${message_aggregator_CXXSRCS}
-  ${message_aggregator_RESOURCES_RCC}
-  ${message_aggregator_VERSION_RESOURCES}
-  )
-target_link_libraries (message_aggregator wsjt_qt Qt5::Widgets wsjtx_udp-static)
-
-if (WSJT_CREATE_WINMAIN)
-  set_target_properties (message_aggregator PROPERTIES WIN32_EXECUTABLE ON)
-endif (WSJT_CREATE_WINMAIN)
-
-if (UNIX)
-  if (NOT WSJT_SKIP_MANPAGES)
-    add_subdirectory (manpages)
-    add_dependencies (wsjtx manpages)
-  endif (NOT WSJT_SKIP_MANPAGES)
-  if (NOT APPLE)
-    add_subdirectory (debian)
-    add_dependencies (wsjtx debian)
-  endif (NOT APPLE)
-endif (UNIX)
-
 #
 # installation
 #
-install (TARGETS wsjtx
-  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT runtime
-  BUNDLE DESTINATION . COMPONENT runtime
-  )
 
 # install (TARGETS wsjtx_udp EXPORT udp
 #   RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
@@ -1577,12 +1379,7 @@
 #   DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/wsjtx
 #   )
 
-install (TARGETS udp_daemon message_aggregator wsjtx_app_version
-  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT runtime
-  BUNDLE DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT runtime
-  )
-
-install (TARGETS jt9 wsprd fmtave fcal fmeasure
+install (TARGETS wsjtx_app_version jt9 wsprd
   RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT runtime
   BUNDLE DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT runtime
   )
@@ -1595,38 +1392,6 @@
   )
 endif(WSJT_BUILD_UTILS)  
 
-install (PROGRAMS
-  ${RIGCTL_EXE}
-  DESTINATION ${CMAKE_INSTALL_BINDIR}
-  #COMPONENT runtime
-  RENAME rigctl-wsjtx${CMAKE_EXECUTABLE_SUFFIX}
-  )
-
-install (PROGRAMS
-  ${RIGCTLD_EXE}
-  DESTINATION ${CMAKE_INSTALL_BINDIR}
-  #COMPONENT runtime
-  RENAME rigctld-wsjtx${CMAKE_EXECUTABLE_SUFFIX}
-  )
-
-install (PROGRAMS
-  ${RIGCTLCOM_EXE}
-  DESTINATION ${CMAKE_INSTALL_BINDIR}
-  #COMPONENT runtime
-  RENAME rigctlcom-wsjtx${CMAKE_EXECUTABLE_SUFFIX}
-  )
-
-install (FILES
-  README
-  COPYING
-  AUTHORS
-  THANKS
-  NEWS
-  BUGS
-  DESTINATION ${CMAKE_INSTALL_DOCDIR}
-  #COMPONENT runtime
-  )
-
 install (FILES
   cty.dat
   cty.dat_copyright.txt
@@ -1635,13 +1400,6 @@
   #COMPONENT runtime
   )
 
-install (DIRECTORY
-  example_log_configurations
-  DESTINATION ${CMAKE_INSTALL_DOCDIR}
-  FILES_MATCHING REGEX "^.*[^~]$"
-  #COMPONENT runtime
-  )
-
 #
 # Mac installer files
 #
@@ -1693,22 +1451,6 @@
   "${CMAKE_CURRENT_BINARY_DIR}/wsjtx_config.h"
   )
 
-
-if (NOT WIN32 AND NOT APPLE)
-  # install a desktop file so wsjtx appears in the application start
-  # menu with an icon
-  install (
-    FILES wsjtx.desktop message_aggregator.desktop
-    DESTINATION share/applications
-    #COMPONENT runtime
-    )
-  install (
-    FILES icons/Unix/wsjtx_icon.png
-    DESTINATION share/pixmaps
-    #COMPONENT runtime
-    )
-endif (NOT WIN32 AND NOT APPLE)
-
 if (APPLE)
   set (CMAKE_POSTFLIGHT_SCRIPT
     "${wsjtx_BINARY_DIR}/postflight.sh")
