diff -ur js8call-2.1.1-orig/CMake/Modules/Findhamlib.cmake js8call-2.1.1/CMake/Modules/Findhamlib.cmake
--- js8call-2.1.1-orig/CMake/Modules/Findhamlib.cmake	2020-05-23 15:38:20.730349612 +0000
+++ js8call-2.1.1/CMake/Modules/Findhamlib.cmake	2020-05-23 15:39:28.829772207 +0000
@@ -78,4 +78,4 @@
 # Handle the  QUIETLY and REQUIRED  arguments and set  HAMLIB_FOUND to
 # TRUE if all listed variables are TRUE
 include (FindPackageHandleStandardArgs)
-find_package_handle_standard_args (hamlib DEFAULT_MSG hamlib_INCLUDE_DIRS hamlib_LIBRARIES hamlib_LIBRARY_DIRS)
+find_package_handle_standard_args (hamlib DEFAULT_MSG hamlib_INCLUDE_DIRS hamlib_LIBRARIES)
diff -ur js8call-2.1.1-orig/CMakeLists.txt js8call-2.1.1/CMakeLists.txt
--- js8call-2.1.1-orig/CMakeLists.txt	2020-05-23 15:38:20.730349612 +0000
+++ js8call-2.1.1/CMakeLists.txt	2020-05-23 15:52:46.103389553 +0000
@@ -683,7 +683,7 @@
 #
 # libhamlib setup
 #
-set (hamlib_STATIC 1)
+set (hamlib_STATIC 0)
 find_package (hamlib 3 REQUIRED)
 find_program (RIGCTL_EXE rigctl)
 find_program (RIGCTLD_EXE rigctld)
@@ -1106,20 +1106,6 @@
   BUNDLE DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT runtime
   )
 
-install (PROGRAMS
-  ${RIGCTL_EXE}
-  DESTINATION ${CMAKE_INSTALL_BINDIR}
-  #COMPONENT runtime
-  RENAME rigctl-local${CMAKE_EXECUTABLE_SUFFIX}
-  )
-
-install (PROGRAMS
-  ${RIGCTLD_EXE}
-  DESTINATION ${CMAKE_INSTALL_BINDIR}
-  #COMPONENT runtime
-  RENAME rigctld-local${CMAKE_EXECUTABLE_SUFFIX}
-  )
-
 install (FILES
   README
   COPYING
Only in js8call-2.1.1/: hamlib.patch
